server/order-service/src/main/java/com/ecomproject/order_service/controller/OrderController.java

```diff
@@ -0,0 +1,26 @@
+package com.ecomproject.order_service.controller;
+
+import com.ecomproject.order_service.dto.OrderRequest;
+import com.ecomproject.order_service.service.OrderService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/order")
+public class OrderController {
+
+    private final OrderService orderService;
+
+    @PostMapping
```

Comment:
use this instead -> @ResponseStatus(HttpStatus.CREATED)

Reply:
ok sir adding this